department-of-veterans-affairs/va.gov-cms
# VACMS-8709: remove field_facility_service_hours from service_location…

… paragraph

## Description

Closes #8709 

## Testing done

Visual testing

## Screenshots
<img width="1716" alt="Screen Shot 2022-04-12 at 4 58 16 PM" src="https://user-images.githubusercontent.com/21045418/163053502-20e3d2bf-fe22-441a-ba34-5f200c353b2d.png">


## QA steps

As an admin user:

Visit admin/structure/paragraphs_type/service_location/fields and verify the field_facility_service_hours field has been removed

### Select Team for PR review

- [ ] `Sitewide crew ` (leave Sitewide unchecked and check the specific team instead)
